<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulacionMinima</class>
 <widget class="QDialog" name="SimulacionMinima">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frameVisualizacionFabricas">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>291</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frameBotonesFabricas">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>91</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>95</width>
       <height>379</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnConfigurarFabricaAmor">
        <property name="text">
         <string>Configurar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPausarFabricaAmor">
        <property name="text">
         <string>Pausar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReiniciarFabricaAmor">
        <property name="text">
         <string>Reiniciar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblEstadoFabricaAmor">
        <property name="text">
         <string>Estado : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfigurarFabricaArte">
        <property name="text">
         <string>Configurar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPausarFabricaArte">
        <property name="text">
         <string>Pausar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReiniciarFabricaArte">
        <property name="text">
         <string>Reiniciar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblEstadoFabricaArte">
        <property name="text">
         <string>Estado :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfigurarFabricaOdio">
        <property name="text">
         <string>Configurar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPausarFabricaOdio">
        <property name="text">
         <string>Pausar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReiniciarFabricaOdio">
        <property name="text">
         <string>Reiniciar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblEstadoFabricaOdio">
        <property name="text">
         <string>Estado :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frameImagenesFabricas">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>81</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>61</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnHistorialColaFabricaAmor">
        <property name="styleSheet">
         <string notr="true">image: url(:/Recursos/ImagenesProyecto/archivo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblImgFabricaAmor">
        <property name="styleSheet">
         <string notr="true">image: url(:/Recursos/ImagenesProyecto/factory.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHistorialColaFabricaArte">
        <property name="styleSheet">
         <string notr="true">image: url(:/Recursos/ImagenesProyecto/archivo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblImgFabricaArte">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/Recursos/ImagenesProyecto/factory.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHistorialColaFabricaOdio">
        <property name="styleSheet">
         <string notr="true">image: url(:/Recursos/ImagenesProyecto/archivo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblImgFabricaOdio">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/Recursos/ImagenesProyecto/factory.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frameColaFabricaOdio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblCantidadOdio">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="indent">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblTiempoFabricaOdio">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>t</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameColaFabricaArte">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblCantidadArte">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="indent">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblTiempoFabricaArte">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>t</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameColaFabricaAmor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblCantidadAmor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="indent">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblTiempoFabricaAmor">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>t</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>81</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblSemaforoFabricante">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/semaforo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>41</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lblAmorFabricante">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblArteFabricante">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblOdioFabricante">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblTiempoFabricante">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAleatorioFabricante">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>123</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfigurarFabricante">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Configurar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPausarFabricante">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pausar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReiniciarFabricante">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reiniciar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEstadoFabricante">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Estado :</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>630</y>
     <width>395</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnIniciar">
      <property name="text">
       <string>Iniciar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPausar">
      <property name="text">
       <string>Pausar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnReanudar">
      <property name="text">
       <string>Reanudar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDetener">
      <property name="text">
       <string>Detener</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frameColaBebes">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblCantidadBebes">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="indent">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Recursos/ImagenesProyecto/conveyor.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHistorialColaBebes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Recursos/ImagenesProyecto/archivo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameAlmacen">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblAlmacen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Recursos/ImagenesProyecto/depot.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblCantidadBebesFabricados">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHistorialAlmacen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Recursos/ImagenesProyecto/archivo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_PausarEspecial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Pausar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ReanudarEspecial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reanudar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reiniciarEspecial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="box_TiempoEspecial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pedido Especial</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Historial_Especial">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recursos/ImagenesProyecto/archivo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
